ios/RNReaderSDKAuthorization.m

```diff
@@ -94,6 +94,15 @@ @implementation RNReaderSDKAuthorization
     });
 }
 
+RCT_EXPORT_METHOD(isAuthorizationInProgress
+                  : (RCTPromiseResolveBlock)resolve andRejecter
```

Comment:
can you make the parameter style consistent with other method?

Reply:
I was actually scratching my chin about this for a while.
Why did you ever use `RCT_REMAP_METHOD` ?

That's meant for whenever you have multiple native methods with the same name, which is not the case on your module.

That being said, I can definitely make the change.
Standing by for your thoughts on this.

Reply:
Made the change